app/components/CapitalProjectsList/CapitalProjectsList.tsx

```diff
@@ -0,0 +1,76 @@
+import { Box, Flex, Text, VStack, WrapItem } from "@nycplanning/streetscape";
+import { useState } from "react";
+import { Agency, CapitalProject } from "~/gen";
+import { CapitalProjectsListItem } from "./CapitalProjectsListItem";
+import { formatFiscalYearRange, currentDate } from "../../utils/utils";
+import { Link, useLocation, useSearchParams } from "@remix-run/react";
+
+export interface CapitalProjectsListProps {
+  capitalProjects: CapitalProject[];
+  agencies: Agency[];
+}
+
+export const CapitalProjectsList = ({
+  capitalProjects,
+  agencies,
+}: CapitalProjectsListProps) => {
+  const [isExpanded] = useState(false);
+  const [searchParams] = useSearchParams();
+  const location = useLocation().pathname;
+  const listBody =
+    capitalProjects.length === 0 ? (
+      <Text>Reached end of projects</Text>
+    ) : (
+      capitalProjects.map((capitalProject) => {
+        return (
+          <Box
```

Comment:
Could this `Box` and `Link` be moved into `CapitalProjectsListItem`? It feels like that component would be more self-contained if it included those.

Reply:
At the time I only wanted to pass in the strings that were going to be rendered in `CapitalProjectsListItem` (instead of passing id and managingCode and also calling the searchParams and location hooks) so that the list can handle the logic of assigning links

Reply:
Gotcha. Can you move that into the list item component? I don't think it's an issue to have that component depend on those hooks if it needs them to do its job. You could have that component take in a `capitalProject` (of type `CapitalProjectBudgeted`), instead of passing in strings piecemeal.